packages/boxel-ui/addon/components/field-container/index.gts

```diff
@@ -51,6 +51,67 @@ const FieldContainer: TemplateOnlyComponent<Signature> = <template>
       {{/if}}
     </Tag>
   {{/let}}
+
+  <style>
+    .boxel-field {
+      --boxel-field-label-align: normal;
+      --boxel-field-label-justify-content: normal;
+      --boxel-field-label-padding-top: 0;
+
+      display: grid;
+      gap: var(--boxel-sp-xs) 0;
+    }
+
+    .boxel-field--vertical {
```

Comment:
But there's no reason to keep namespacing all these other selectors with a `.boxel-field` prefix.

There's also no reason to keep the double dashes around. This could be just `.vertical`.